.travis.yml

```diff
@@ -17,13 +17,19 @@ cache:
     directories:
         - $HOME/.composer/cache
 
+before_script:
+    - echo 'phar.readonly = "0"' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+
 before_install:
     - sudo add-apt-repository -y ppa:git-core/ppa
     - sudo apt-get update && sudo apt-get install git-man && sudo apt-get install -y git
     - git --version
     - phpenv config-rm xdebug.ini || true
+    - export PATH="$PATH:$HOME/.composer/vendor/bin"
 
-install: composer update --prefer-dist --no-interaction --optimize-autoloader
+install:
+    - composer global require kherge/box --prefer-dist --no-interaction --optimize-autoloader
```

Comment:
Heh.. There is already a PR https://github.com/box-project/amend/pull/11/files, let's move on  :/